examples/visualization/plot_evoked_arrowmap.py

```diff
@@ -0,0 +1,30 @@
+"""
+========================================
+Plotting topographic arrowmaps of evoked data
+========================================
```

Comment:
you need as many = as the number of characters for it to render fine

Reply:
Done